docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md

```diff
@@ -0,0 +1,177 @@
+	- `lighting_space_lookup["GYMNASIUM_FITNESS_CENTER_EXERCISE_AREA"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["GYMNASIUM_FITNESS_CENTER_PLAYING_AREA"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_EMERGENCY_ROOM"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_EXAM_TREATMENT_ROOM"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_NURSERY"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_NURSES_STATION"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_OPERATING_ROOM"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_PATIENT_ROOM"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_PHYSICAL_THERAPY_ROOM"] = "HOSPITAL"`
+	- `lighting_space_lookup["HEALTHCARE_FACILITY_RECOVERY_ROOM"] = "HOSPITAL"`
+	- `lighting_space_lookup["LIBRARY_READING_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["LIBRARY_STACKS"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_DETAILED_MANUFACTURING_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_EQUIPMENTROOM"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_EXTRA_HIGH_BAY_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_HIGH_BAY_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MANUFACTURING_FACILITY_LOW_BAY_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["MUSEUM_GENERAL_EXHIBITION_AREA"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["MUSEUM_RESTORATION_ROOM"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["POST_OFFICE_SORTING_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["RELIGIOUS_FACILITY_FELLOWSHIP_HALL"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["RELIGIOUS_FACILITY_WORSHIP_PULPIT_CHOIR_AREA"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["RETAIL_FACILITIES_DRESSING_FITTING_ROOM"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["RETAIL_FACILITIES_MALL_CONCOURSE"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_I_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_II_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_III_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["SPORTS_ARENA_PLAYING_AREA_CLASS_IV_FACILITY"] = "PUBLIC_ASSEMBLY"`
+	- `lighting_space_lookup["TRANSPORTATION_FACILITY_BAGGAGE_CAROUSEL_AREA"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["TRANSPORTATION_FACILITY_AIRPORT_CONCOURSE"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["TRANSPORTATION_FACILITY_TICKET_COUNTER"] = "OTHER_NON_RESIDENTIAL"`
+	- `lighting_space_lookup["WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS"] = "HEATED-ONLY_STORAGE"
+	- `lighting_space_lookup["WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS"] = "HEATED-ONLY_STORAGE"
+

```

Comment:
We usually don't include look-up tables in the functions or RDSs (e.g., do not have lookup for U-factors depending on building type and climate zone). But perhaps it's justified here since this is not coming from 90.1. @weilixu what do you think?

Reply:
the concern was what if there is no HVAC BAT entered at the building_segment level.  Christina provided mapping of lighting building area types to HVAC BAT (I think this was provided by ASHRAE).  I took a shot at extending the mapping to lighting space types as most of them are sub-categories of lighting building space types.

We absolutely need an HVAC BAT, so the other option would be to make it a required field in the schema - thoughts @JasonGlazer ?

@weilixu - I see your point about storage can be cooled.  G3.1.1.f takes care of this "If the baseline HVAC system type is 9 or 10, use additional system types for all HVAC zones that are mechanically cooled in the proposed design."

Reply:
@KarenWGard I don't think HVAC BAT should be a required part of schema because it will mean that files that don't have this specified are rejected by the RCT. Even if the HVAC-related rules can't be checked, RCT can still be used to check envelope and lighting rules.

Reply:
> Generally the approach I think is ok, but I am not sure if RCT should provide such mapping since it is not part of Standard, and it is not entirely clear how some of the lighting spaces can be mapped to HVAC building type

@weilixu I agree, but perhaps if we based HVAC building area type on lighting space type mapping we can have a note to this effect as part of outcome, to suggest that reviewer manually double-check the HVAC building area type.

Reply:
@mkarpman - I agree.  I think that if we don't do the mapping, but we require HVAC BAT, the simulation software developers will do their own mapping, and it will be inconsistent.

Reply:
So I think the plan is to do the mapping. For transparency, let's also include the mapping into documentation outside of the RDSs. Perhaps it should be included with the 229 package. Perhaps also make it an exterinal library table similar to how ASHRAE tables are handled, so that it is not incapsulated into the RDS.

Reply:
function has been updated with mapping.  Description was updated with a clearer description of how mapping happens.  Mapping re-written to be more easily readable.  I've noted where I am unclear on what mapping is correct.  Please review where noted.

Support spaces are initially categorized as `OTHER_UNDETERMINED` and then re-mapped to the predominant space area type at the end of the function.  Please review what is mapped as `OTHER_UNDETERMINED`

Reply:
@weilixu will move this out when the RDS is implemented and update the RDS to not have this mapping